lib/services/npq/output_payments.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Services
+  module Npq
+    class OutputPayment
+      include InitializeWithConfig
+      delegate :retention_points, :per_participant_price, to: :config
+
+      def call
+        retention_points.transform_values do |values|
+          {
+            per_participant: per_participant_output_payments,
+            total_output_payments: total_output_payments(values[:retained_participants]),
+          }
+        end
+      end
+
+    private
+
+      def output_payment_split
+        1.0 / retention_points.length
+      end
+
+      def per_participant_output_payments
+        (per_participant_price * 0.6 * output_payment_split).round(2)
+      end
+
+      def total_output_payments(retained_participants)
```

Comment:
singular? Is it a total "Output payment" that seems to be consistent with how it's used in other places.